configs/vision/dino_vit/offline/bach.yaml

```diff
@@ -39,6 +39,7 @@ trainer:
                 pretrained: ${oc.env:PRETRAINED, true}
                 force_reload: ${oc.env:FORCE_RELOAD, false}
               checkpoint_path: ${oc.env:CHECKPOINT_PATH, null}
+          overwrite: false
```

Comment:
maybe lets remove the `overwrite` here, since the default should be false and to not overpopulate the config

Reply:
the default used to be `False`, seems that at some point this got changed (maybe by accident), which is why I now defined this explicitely

Reply:
I see, lets keep it them :D